src/status_im2/contexts/syncing/view.cljs

```diff
@@ -4,51 +4,70 @@
             [quo2.foundations.colors :as colors]
             [react-native.core :as rn]
             [status-im2.contexts.syncing.sheets.sync-device-notice.view :as sync-device-notice]
-            [status-im2.contexts.syncing.styles :as styles]
+            [status-im2.contexts.syncing.style :as style]
             [utils.re-frame :as rf]))
+;;TODO remove mock data (#https://github.com/status-im/status-mobile/issues/15142)
```

Comment:
This comment seems outdated, since the issue link is pointing to `Allow user to sync devices` and not something related to removing mock data. Or even better it looks like we could remove this TODO.

Reply:
that story is like the larger story which covers all of this flow. So as part of that I will remove the mock data. It will be adjusted very soon in a follow up!

Reply:
Okay, nice!